app/icon_set.py

```diff
@@ -121,5 +121,6 @@ def serialize(self):
             "colorable": self.colorable,
             "icons_url": self.get_icons_url(),
             "template_url": get_icon_set_template_url(get_base_url()),
-            "has_description": bool(find_descripton_file(self.name))
+            "has_description": bool(find_descripton_file(self.name)),
+            "language": self.name.split('-')[-1] if '-' in self.name else False
```

Comment:
Not sure, if this logic will do for all cases.
For example a file called "001-Beschaedigung.png" would lead to a wrong language.

Reply:
We might take the language from the directory or the icon-set name respectively.
In that case we need to make sure, that the language is always part of the icon set name.
Why do we need the language in the first place?

Reply:
this is for the icon set, not the icons

Reply:
we need the language so we can properly translate the icon set selector dropdown on the mapviewer

Reply:
Ah, stupid me :see_no_evil: 
Of course, that makes sense then!

Reply:
In this case, we need to document and make sure, that all (new) icon sets are always named correctly and don't include `-` unless for adding a language.

Reply:
That touches our architectural decissions @boecklic

Reply:
I can also use a dictionary if that is preferable

Reply:
I have replaced it with a dictionary for now, we can change it back later if is the preferred solution

Reply:
Thanks! I had a similar idea later yesterday evening.
Slightly diferent and a bit more generic, without having to hardcode the icon-set's languages.

We could have a list with the languages we support, i.e. `SUPPORTED_LANGUAGES = ["FR", "IT", "RM", "DE", "EN"]` for example.
Then we can use your approach you had in a few commits before, i.e. trying to get the icon-set's language from the name/path. Then check, if the derived language is in our list of supported languages.
If it is: bingo, else `None` for the language.
How about that?